portfolio/src/main/webapp/index.html

```diff
@@ -10,9 +10,20 @@
     <div id="content">
       <h1>My Portfolio</h1>
       <p>This is my portfolio.</p>
+      <p>"Lorem ipsum dolor sit amet, consectetur adipiscing elit,
```

Comment:
What is the goal of this paragraph? Consider replacing paragraph with something meaningful, maybe something about yourself, what you enjoy, etc. The cat picture below is a great idea!